spk/salt-minion/Makefile

```diff
@@ -1,41 +1,73 @@
 SPK_NAME = salt-minion
-SPK_VERS = 3001.1
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+SPK_VERS = 3003.2
+else
+SPK_VERS = 3000.9
+endif
 SPK_REV = 1
 SPK_ICON = src/salt-minion.png
 
-PIP = $(WORK_DIR)/../../../native/python3/work-native/install/usr/local/bin/pip
-BUILD_DEPENDS = cross/python3
-SPK_DEPENDS = "python3>=3.7"
-UNSUPPORTED_ARCHS = $(PPC_ARCHS) $(ARMv5_ARCHS)
+DEPENDS = cross/patch
+
+# DSM 6 has python2, DSM 7 has python3
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+BUILD_DEPENDS  = cross/python38 cross/setuptools cross/pip
+else
+BUILD_DEPENDS  = cross/python2 cross/setuptools_py2 cross/pip_py2
+SPK_DEPENDS = "python>=2.7.11"
+endif
+BUILD_DEPENDS += cross/wheel
+BUILD_DEPENDS += cross/salt
+
+include ../../mk/spksrc.archs.mk
 
 # Requirements file generation
 # /usr/local/python3/bin/python3 -mvirtualenv --python=python3  salt-env
 # virtualenv --python=python3  salt-env
 # source salt-env/bin/activate
 # pip install salt
 # pip freeze > requirements.txt
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+ifeq ($(findstring $(ARCH),$(ARMv5_ARCHS)),$(ARCH))
+WHEELS = src/requirements_gcc46.txt
+else
 WHEELS = src/requirements.txt
+endif
+else
+WHEELS = src/requirements_py2.txt
+endif
 
 MAINTAINER = SynoCommunity
 DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
 RELOAD_UI = yes
 DISPLAY_NAME = Salt Minion
-CHANGELOG = "Update to 3001.1"
+ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
+CHANGELOG = "Update to 3003.2"
+else
+CHANGELOG = "Last version for Python2 on DSM 6.x is 3000.9"
+endif
 
 HOMEPAGE   = https://www.saltstack.com/
-LICENSE    = Apache
```

Comment:
You can have both the `LICENSE_FILE` and `LICENSE = Apache`. Please keep it anyway.

Reply:
Ok, I prefer to have a LICENSE file that (on DSM 7) pop up and you need to accept the license to install the package.

On that note. Shouldn't all licenses be followed by a SynoCommunity disclaimer that the package is supported as-is and SynoCommunity will not take any responsibility for any data loss ?

Reply:
Yes you can keep the file :+1:  All I'm saying is to add the `LICENSE = Apache` variable back, so you end up with both.